integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataRecordSet.java

```diff
@@ -79,30 +81,27 @@ private Expression parseConstraint2Expression(TupleDomain<ColumnHandle> constrai
   @Override public RecordCursor cursor() {
     List<TableBlockInfo> tableBlockInfoList = new ArrayList<TableBlockInfo>();
 
-    tableBlockInfoList.add(new TableBlockInfo(split.getLocalInputSplit().getPath().toString(),
+    tableBlockInfoList.add(new TableBlockInfo(split.getLocalInputSplit().getPath(),
         split.getLocalInputSplit().getStart(), split.getLocalInputSplit().getSegmentId(),
         split.getLocalInputSplit().getLocations().toArray(new String[0]),
         split.getLocalInputSplit().getLength(), new BlockletInfos(),
         //blockletInfos,
         ColumnarFormatVersion.valueOf(split.getLocalInputSplit().getVersion()), null));
-    queryModel.setTableBlockInfos(tableBlockInfoList);
 
+    queryModel.setTableBlockInfos(tableBlockInfoList);
     queryExecutor = QueryExecutorFactory.getQueryExecutor(queryModel);
 
-    try {
+    CarbonProperties.getInstance().addProperty("carbon.detail.batch.size",
```

Comment:
I can't find where code use this property(carbon.detail.batch.size)? don't know need to add property at here ?